locales/templates/app.pot

```diff
@@ -635,3 +654,37 @@ msgctxt "dialogue_ocr_share_description"
 msgid "Find screenshot-to-text useful? Share it now!"
 msgstr ""
 
+#. String displayed to inform users search is performing
+msgctxt "search_transition_searching"
+msgid "Searching…"
+msgstr ""
+
+msgctxt "search_error_title_noresult"
+msgid "No results found"
+msgstr ""
+
+msgctxt "search_error_content_try"
+msgid "Try another word"
+msgstr ""
+
+#. Parameter %1$d is the quantity of matched search results, i.e. 100 results
+#, c-format
+msgctxt "search_separator_results"
+msgid "%1$d results"
```

Comment:
I know Android has plural support, but compare-locales and Pontoon don't.
So I believe a workaround to this would be to change the string and make it like:
"Results: %1$d"

Reply:
From a UX perspective, "X results" is more intuitive and understandable compared to the workaround. Based on our previous discussion, the decision was to keep it plural since ScreenshotGo is only available in Indonesia, where has no such concern. Once we decide to spread to other countries, we'll implement the Android plural support.

Reply:
Ok let's leave as is then since this is for Indonesia only. We should keep note of this though in case ScreenshotGO expands in the future. Thanks!